src/renderer/components/Button/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import MUIButton, { ButtonProps as MUIButtonProps } from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import React, { FC } from 'react';
+import './Button.scss';
+
+interface ButtonProps extends MUIButtonProps {
+  textCase?: 'lowercase' | 'uppercase' | 'sentence-case';
+  fontWeight?: string;
+  textColor?: any;
+}
+
+const Button: FC<ButtonProps> = ({
+  textCase,
+  fontWeight,
+  textColor,
+  children,
+  ...rest
+}) => {
+  return (
+    <MUIButton className={`${textCase ?? 'lowercase'}`} {...rest}>
```

Comment:
You can do `<MUIButton className={textCase} {...rest}>` and assign default props to the component. (eslint also notified about this).

```ts

Button.defaultProps = {
  textCase: 'lowercase',
  fontWeight: 'normal',
  textColor: 'text.secondary',
};
```